src/javascripts/components/code-blocks.mjs

```diff
@@ -0,0 +1,50 @@
+/**
+ * Responsive behaviour for code blocks
+ */
+class CodeBlocks {
```

Comment:
**nitpick** Would we need the same behaviour of removing tabindex when not overflowing for any element we anticipate having to scroll or is it really specific to code blocks? If the former, a more generic name (`ScrollContainer` maybe) would save a future renaming.